sched/worker/fake/fake.go

```diff
@@ -13,35 +14,22 @@ func (c *noopWorkerController) RunAndWait(task sched.TaskDefinition) error {
 	return nil
 }
 
-func NewNoopWorker() WorkerController {
+func NewNoopWorker() worker.WorkerController {
 	return &noopWorkerController{}
 }
 
-type noopWorkerWrapper struct{}
-
-func (w *noopWorkerWrapper) Wrap(node cluster.Node) cluster.Node {
-	return &workerNode{
-		worker: &noopWorkerController{},
-		node:   node,
-	}
-}
-
-var NoopWorkerWrapper = &noopWorkerWrapper{}
-
-type waitingWorkerWrapper struct {
+func NoopWorkerFactory(node cluster.Node) worker.WorkerController {
```

Comment:
I know we refer to these as factories in conversation, but we also had an earlier discussion where we replaced '*Factory' with 'New*' or 'Make*'. Probably don't want to have all three.